website/static/js/folderpicker.js

```diff
@@ -192,13 +201,15 @@ function FolderPicker(selector, opts) {
 $.fn.folderpicker = function(options) {
     this.each(function() {
         // Treebeard must take an ID as a selector if using as a jQuery plugin
-        if (!this.id) { throw 'FolderPicker must have an ID if initializing with jQuery.'; }
+        if (!this.id) {
+            throw 'FolderPicker must have an ID if initializing with jQuery.';
```

Comment:
Always throw `Error` objects rather than strings. This is more readable and allows Raven to capture the error: https://raven-js.readthedocs.org/en/latest/usage/index.html#how-to-actually-capture-an-error-correctly